repondeur/zam_repondeur/static/js/dossiers.js

```diff
@@ -20,18 +20,42 @@ application.register(
   'dossier-invite',
   class extends Stimulus.Controller {
     clean(event) {
+      const textarea = event.target
+      const initial = textarea.value.trim()
+
       /* https://stackoverflow.com/a/52078216 */
       const reg = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/g
       const paste = (event.clipboardData || window.clipboardData).getData(
         'text'
       )
+
+      const separators = [';', ',', '\n']
       let emails = []
-      ;[';', ',', '\n'].forEach(separator => {
+      separators.forEach(separator => {
         if (paste.includes(separator)) {
-          emails = paste.split(separator).map(line => line && line.match(reg))
+          emails = paste
+            .split(separator)
+            .map(line => line && line.match(reg))
+            .filter(s => s)
```

Comment:
What is that line doing?

Reply:
See commit message: the goal is to remove nulls and empty strings produced by the processing